src/services/api.ts

```diff
@@ -6,14 +6,14 @@ const API_ROOT = 'http://localhost:8080'
 let request = (method: 'GET' | 'POST') =>
   (url: string) =>
     <T>(body: object) =>
-      fetch(new Request(API_ROOT + url, {
```

Comment:
Redundant, and was one more thing we would have needed to polyfill to run tests in NodeJS using Jest.